Scripts/script-DataURLReputation.yml

```diff
@@ -4,19 +4,26 @@ commonfields:
 name: DataURLReputation
 script: |-
   var score = -1;
-  var rep = executeCommand('url', {url: args.input});
-  var scores = dq(rep, 'EntryContext.DBotScore.Score');
-  for (var i = 0; scores && i < scores.length; i++) {
-    if (scores[i] && score < scores[i]) {
-      score = scores[i];
-    }
+  var res = executeCommand("findIndicators", {"query": "type:URL and value:" + args.input})
```

Comment:
add `manualScore:T` to filter only for manual score
wrap value with a single bracket `'`